internal/proto/local/controller/api/resources/targets/v1/target.proto

```diff
@@ -43,8 +43,8 @@ message SessionCredential {
 	// Output only. The library which generated this credential.
 	CredentialLibrary credential_library = 1;
 
-	// Output only. The secret of this credential.
-	google.protobuf.Value secret = 2;
+	// Output only. The secret of this credential base64 encoded.
+	string secret = 2;
```

Comment:
Why not make this `bytes`? The result of nearly any marshaling function will be `[]byte` and then we get the base64 encoding for free, and on the other end we get the decoding for free.

Reply:
In my branch working on some enhancements I've made this change; see what you think when I PR.

Reply:
OK, I've come around to this; having it as a string means more easily replacing it on the client side without then having it auto marshaled to base64.